lib/plugins/aws/provider/awsProvider.js

```diff
@@ -146,11 +146,21 @@ class AwsProvider {
         f()
           .then(resolve)
           .catch((e) => {
-            if (e.statusCode === 429) {
+            const err = e;
+            if (err.statusCode === 429) {
               that.serverless.cli.log("'Too many requests' received, sleeping 5 seconds");
               setTimeout(doCall, 5000);
             } else {
-              reject(e);
+              if (err.message === 'The security token included in the request is invalid') {
```

Comment:
Could you change this to match via a regex. Something like

```javascript
if (err.message.match(/.+security token.+is invalid/)) {
  // ...
}
```

This way we would just check for the most important parts of the error message and it would be more future-proof if AWS decides to change the message later on.